doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -257,7 +257,7 @@ nRF Desktop
     The value is now aligned with the Fast Pair requirements.
   * Enabled the :ref:`CONFIG_DESKTOP_CONFIG_CHANNEL_OUT_REPORT <config_desktop_app_options>` Kconfig option for the nRF Desktop peripherals with :ref:`nrf_desktop_dfu`.
     The option mitigates HID report rate drops during DFU image transfer through the nRF Desktop dongle.
-    The output report is also enabled for the ``nrf52kbd_nrf52832`` build target in the debug configuration to maintain consistency with the release configuration.
+    The output report is also enabled for the ``nrf52kbd_nrf52832`` board target in the debug configuration to maintain consistency with the release configuration.
```

Comment:
```suggestion
    The output report is also enabled for the ``nrf52kbd/nrf52832`` board target in the debug configuration to maintain consistency with the release configuration.
```

(are these latest rel notes?)

Reply:
These are latest rel notes, not yet released. I'll fix.